### PR TITLE
feat(dashboard): change the update button to migrate

### PR DESCRIPTION
**Pull request by sachinrameshn**
| Question         | Answer
| ---------------- | ---
| Branch?          | `develop`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #MANAGER-9078
| License          | BSD 3-Clause

- [X] Try to keep pull requests small so they can be easily reviewed.
- [X] Commits are signed-off
- [X] Only FR translations have been updated
- [X] Branch is up-to-date with target branch
- [X] Lint has passed locally
- [X] Standalone app was ran and tested locally
- [X] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits -- N/A

## Description

Change the upgrade button to migrate and also underlying link

### CODE REVIEW

File: packages/manager/modules/vps/src/dashboard/vps-dashboard.html [hunk 1]
```diff
@@ -126,15 +126,17 @@
                 </div>
                 <div
                     class="oui-tile__item oui-tile__item_button"
-                    data-ng-if="$ctrl.hasFeature($ctrl.DASHBOARD_FEATURES.upgrade) && $ctrl.actions.upgrade.state && $ctrl.actions.upgrade.isAvailable() && !$ctrl.isMigrating"
+                    data-ng-if="$ctrl.canBeMigrated()"
```
Comment (marie-j) [inline packages/manager/modules/vps/src/dashboard/vps-dashboard.html:129]: already exists and we forgot to use it ?

**Review by radireddy — APPROVED** [review]

File: packages/manager/modules/vps/src/dashboard/vps-dashboard.html (hunk 1 above)
Comment (sachinrameshn) [reply packages/manager/modules/vps/src/dashboard/vps-dashboard.html:129]: Yes Marie it was existing one I just used the same to show the migrate option.

File: packages/manager/modules/vps/src/dashboard/vps-dashboard.html [hunk 2]
```diff
@@ -126,15 +126,17 @@
...
                 >
                     <a
-                        class="oui-tile__button oui-button oui-link_icon oui-button_ghost oui-button_block"
-                        data-ui-sref="{{ $ctrl.actions.upgrade.state + $ctrl.constructor.getActionStateParamString($ctrl.actions.upgrade.stateParams) }}"
-                        rel="noopener"
+                        class="oui-tile__button oui-button oui-link_icon"
+                        data-ng-href="{{ $ctrl.migrationLink }}"
```
Comment (mohammed-zahaf) [inline packages/manager/modules/vps/src/dashboard/vps-dashboard.html:133]: Use one time binding to save performance.
```suggestion
                        data-ng-href="{{:: $ctrl.migrationLink }}"
```

**Review by anooparveti — COMMENTED** [review]
**suggestion:** If the upgrade text is not used anymore, it can be removed from the controller.

File: packages/manager/modules/vps/src/dashboard/vps-dashboard.html (hunk 2 above)
Comment (sachinrameshn) [reply packages/manager/modules/vps/src/dashboard/vps-dashboard.html:133]: Okay Zahaf will do the modification.

**Review by mohammed-zahaf — DISMISSED** [review]

**Review by anooparveti — DISMISSED** [review]

**Review by JeremyDec — APPROVED** [review]

**Review by anooparveti — APPROVED** [review]

**Review by ghyenne — APPROVED** [review]

**Pull request merged**